ldk/node/src/hostClients/networkService.ts

```diff
@@ -2,11 +2,14 @@ export interface HttpRequest {
   url: string;
   method: string;
   body: string | Uint8Array;
+
+  headers: Record<string, string>
 }
 
 export interface HttpResponse {
   statusCode: number;
   data: string | Uint8Array;
+  headers: Record<string, Array<string>>;
```

Comment:
I'm wondering if it'd make more sense to return the same header value type as we sent (a single string with comma separated values instead of an array of values).

Reply:
We went the other way with it, but great callout.